android/app/src/main/java/com/shockwallet/NotificationService.java

```diff
@@ -377,7 +391,7 @@ public void onResponse( Response response) throws IOException {
                         NotificationService.ApiPubKey = resJson.getString("APIPublicKey");
 
                         //mSocket = IO.socket("http://"+NotificationService.ip+"?x-shockwallet-device-id=7601a723-b6d4-4020-95a6-6113fb40e2f8");
-                        mSocket = IO.socket("http://"+NotificationService.ip+"?x-shockwallet-device-id="+deviceId);
+                        mSocket = IO.socket("http://"+NotificationService.ip+"?x-shockwallet-device-id="+deviceId+"&IS_LND_SOCKET=true");
```

Comment:
We changed sockets now? No more message notifications but we get invoiced notifications now?

Reply:
Yes, a few weeks ago Emad made the LND socket to be independent of the other socket we have, IIRC the objective was to reimplement the socket from the beginning, test it, and then move the rest of the stuff to this new socket, I might not remember correctly tho, anyways, invoices and transaction are now sent via this new socket